cpg_workflows/workflow.py

```diff
@@ -1289,54 +1284,18 @@ def queue_for_multicohort(self, multicohort: MultiCohort) -> dict[str, StageOutp
         Plug the stage into the workflow.
         """
         output_by_target: dict[str, StageOutput | None] = dict()
-
-        if not (cohorts := multicohort.get_cohorts()):
+        if not (active_sgs := multicohort.get_sequencing_groups()):
+            all_sgs = len(multicohort.get_sequencing_groups(only_active=False))
             logging.warning(
-                f'{len(multicohort.get_cohorts())}/'
-                f'{len(multicohort.get_cohorts(only_active=False))} '
-                f'usable (active=True) cohorts found in the multicohort. Check that '
-                f'`workflow.input_cohorts` is provided, and not all cohorts are skipped ',
+                f'{len(active_sgs)}/{all_sgs} usable (active=True) SGs found in the multicohort. '
+                'Check that input_cohorts` or `input_datasets` are provided and not skipped',
             )
             return output_by_target
 
-        for cohort in cohorts:
-            if not (datasets := cohort.get_datasets()):
-                logging.warning(
-                    f'{len(cohort.get_datasets())}/'
-                    f'{len(cohort.get_datasets(only_active=False))} '
-                    f'usable (active=True) datasets found in the cohort. Check that '
-                    f'`workflow.input_datasets` or `workflow.input_cohorts` is provided, and each cohort is populated in metamist.',
-                )
-                continue
-
-            for dataset in datasets:
-                if not dataset.get_sequencing_groups():
-                    logging.warning(
-                        f'{dataset}: '
-                        f'{len(dataset.get_sequencing_groups())}/'
-                        f'{len(dataset.get_sequencing_groups(only_active=False))} '
-                        f'usable (active=True) sequencing groups found. Check logs above for '
-                        f'possible reasons sequencing groups were skipped (e.g. all sequencing groups ignored '
-                        f'via `workflow.skip_sgs` in config, or they all missing stage '
-                        f'inputs and `workflow.skip_sgs_with_missing_input=true` is set)',
-                    )
-                    continue
```

Comment:
This drops the various logging statements - instead of 'get cohort, did the cohort have contents', then 'get dataset, did the dataset have contents', we're really only interested in SGs. During setup we already fail if the Cohort queries have no returned SGs, so I don't think these are ever hit